### PR TITLE
DOCSUP-2806: Add meta header in RU

### PR DESCRIPTION
**Pull request by ka1bi4**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Detailed description / Documentation draft:

- Update meta information for creating toc.

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]

File: docs/ru/sql-reference/statements/create/row-policy.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-toc_priority: 7
-toc_title: Политика доступа
+toc_priority: 8
+toc_title: "\u041f\u043e\u043b\u0438\u0442\u0438\u043a\u0430\u0020\u0434\u043e\u0441\u0442\u0443\u043f\u0430"
```
Comment (alexey-milovidov) [inline docs/ru/sql-reference/statements/create/row-policy.md:3]: Why?

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]

File: docs/ru/sql-reference/statements/create/row-policy.md (hunk 1 above)
Comment (ka1bi4) [reply docs/ru/sql-reference/statements/create/row-policy.md:3]: Sorry, I wanted to move the Overview section at the beginning of section. It looks like I was overworked and made a mistake.

**Comment by alexey-milovidov** [comment]
I'm about \u escape seqences - looks like some automated change, it makes text less readable.

**Comment by alexey-milovidov** [comment]
@BayoNet Why we have put unreadable \u sequences in docs?

**Pull request merged**